configs/defs/rocky.textproto

```diff
@@ -0,0 +1,1179 @@
+      "Defaults env_reset\n"
+      "References:\n"
+      "1. https://www.sudo.ws/man/1.9.0/sudoers.man.html"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{"
+      "         path:\"/etc/sudoers\""
+      "       }"
+      "      }"
+      "      files_to_check:{files_in_dir:{"
+      "         dir_path:\"/etc/sudoers.d/\""
+      "       }"
+      "      }"
+      "    content_entry:{"
+      "        match_type: NONE_MATCH"
+      "        match_criteria: {"
+      "          filter_regex: \"Authentication timestamp timeout:.*\""
+      "          expected_regex: \"Authentication timestamp timeout:\\\\s*(\\\\d+)\""
+      "          group_criteria:{"
+      "            group_index: 1"
+      "            type: LESS_THAN"
+      "            const: 16"
+      "          }"
+      "        }"
+      "      }"
+      "    }"
+      "  }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "custom-authselect-profile-is-used"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure custom authselect profile is used"
+    description:
+      "A custom profile can be created by copying and customizing one of the default profiles. The "
+      "default profiles include: sssd, winbind, or the nis. This profile can then be customized to "
+      "follow site specific requirements.\n"
+      "You can select a profile for the authselect utility for a specific host. The profile will be "
+      "applied to every user logging into the host."
+    rationale:
+      "A custom profile is required to customize many of the pam options.\n"
+      "When you deploy a profile, the profile is applied to every user logging into the given host"
+    remediation:
+      "Run the following command to create a custom authselect profile:\n"
+      "authselect create-profile <custom-profile name> <options>\n"
+      "Example:\n"
+      "authselect create-profile custom-profile -b sssd --symlink-meta\n"
+      "Run the following command to select a custom authselect profile:\n"
+      "authselect select custom/<CUSTOM PROFILE NAME> {with-<OPTIONS>}\n"
+      "Example:\n"
+      "# authselect select custom/custom-profile with-sudo with-faillock without-nullok\n"
+      "References:\n"
+      "1. authselect(8)"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/etc/authselect/authselect.conf\"}}"
+      "    content_entry:{"
+      "      match_type: ALL_MATCH_ANY_ORDER"
+      "      match_criteria: {"
+      # Expect some kind of policy in the file (i.e. not just comments).
+      "        filter_regex: \"custom/.*\""
```

Comment:
You might need to add .* before - the example shows `- custom/custom-profile Enable SSSD...` as the result string. Has this been tested on a RockyLinux instance?